src/main/java/net/mcreator/ui/dialogs/tools/PackMakerToolUtils.java

```diff
@@ -46,4 +50,20 @@ public static void addGeneratableElementToWorkspace(Workspace workspace, FolderE
 		}
 	}
 
+	public static void addTagEntries(Workspace workspace, TagType tagType, String tagName, String... entries) {
+		// Create tag if it doesn't exist yet
+		TagElement tag = new TagElement(tagType, tagName);
+		if (!workspace.getTagElements().containsKey(tag)) {
+			workspace.addTagElement(tag);
+		}
+
+		// Add entries if they're not already contained in the tag (in normal or managed form)
+		ArrayList<String> tagEntries = workspace.getTagElements().get(tag);
+		for (String entry : entries) {
+			if (!tagEntries.contains(entry) && !tagEntries.contains("~" + entry)) {

```

Comment:
I suggest using TagElement.makeEntryManaged, just for the sake of only handling this prefix in one place